pset-functions.js

```diff
@@ -136,6 +177,43 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+function scrabbleScore(word){
+const letters = word.split("")
+const onePoint = ["a","e","i","o","u","l", "n", "r", "s", "t"] 
+const twoPoints = ["d","g"] 
+const threePoints = ["b","c","m","p"] 
+const fourPoints = ["f","h","v","w","y"] 
+const fivePoints = ["k"] 
+const eightPoints = ["j","x"] 
+const tenPoints = ["q","z"] 
+
+let finalScore = 0
+
+letters.forEach((letter) => {
+  if(onePoint.includes(letter)){
+    finalScore+=1
+  }else if (twoPoints.includes(letter)){
+    finalScore+=2
+  }else if (threePoints.includes(letter)){
+    finalScore+=3
+  }else if (fourPoints.includes(letter)){
+    finalScore+=4
+  }else if (fivePoints.includes(letter)){
+    finalScore+=5 
+  }else if (eightPoints.includes(letter)){
+    finalScore+=8
+  }else if (tenPoints.includes(letter)){
+    finalScore+=10
+  }
+})
+
+console.log(finalScore)
+}
+
+scrabbleScore("javascript")
```

Comment:
@Tambini in this case, the best practice would be to have the function return a final score `return finalScore` as opposed to using `console.log(finalScore)

this makes the function highly reusable. You still have the option of printing the value to the console *outside* of the function 

`console.log(scrabbleScore('javascript'))`

See the solution for more details:  https://github.com/jsd20191008/hw-01-functions-solution/blob/master/pset-functions.js#L178